robot/demos/RemoteArmControl/SingleFile.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
```

Comment:
Not sure what the point is here (or rather if it was achieved, based off the name of the file)

Reply:
I was making a script that doesn't have any udp comms between 2 scripts in order to test things out.